pkg/sentry/fsimpl/fuse/connection.go

```diff
@@ -276,8 +276,15 @@ func (conn *Connection) IsInitialized() bool {
 	return atomic.LoadInt32(&(conn.initialized)) != 0
 }
 
+// Marshallable defines the Marshallable interface for serialize/deserializing
+// FUSE packets.
+type Marshallable interface {
```

Comment:
Is this required? I don't think it's a recommended practice to define interface everywhere and I think @ridwanmsharif had better solution

Reply:
Implemented in https://github.com/google/gvisor/pull/3520